pkg/catalog/fake.go

```diff
@@ -52,23 +52,17 @@ func NewFakeMeshCatalog(kubeClient kubernetes.Interface) *MeshCatalog {
 	mockIngressMonitor.EXPECT().GetIngressResources(gomock.Any()).Return(nil, nil).AnyTimes()
 	mockIngressMonitor.EXPECT().GetAnnouncementsChannel().Return(testChan).AnyTimes()
 
-	// Monitored namespaces is made a set to make sure we don't repeat namespaces on mock
-	listExpectedNs := tests.GetUnique([]string{
-		tests.BookstoreService.Namespace,
-		tests.BookbuyerService.Namespace,
-		tests.BookstoreApexService.Namespace,
-	})
-
 	// #1683 tracks potential improvements to the following dynamic mocks
 	mockKubeController.EXPECT().ListServices().DoAndReturn(func() []*corev1.Service {
 		// play pretend this call queries a controller cache
 		var services []*corev1.Service
 
-		for _, ns := range listExpectedNs {
-			svcList, _ := kubeClient.CoreV1().Services(ns).List(context.Background(), metav1.ListOptions{})
-			for _, svcItem := range svcList.Items {
-				services = append(services, &svcItem)
-			}
+		// Things worked before because internally, the APIs were calling kubeClient APIs with "namespace"
```

Comment:
This comment is not very helpful. Once this is merged `Things worked before because...` lacks context.

Reply:
I left it for review purposes mostly. I imagine that if I stayed with the "assumption" part, reviews might have asked "why change it".

Reply:
will change it to just leave the assumption part, should be explanatory enough